web/src/app/users/UserContactMethodDeleteDialog.js

```diff
@@ -13,7 +13,8 @@ const mutation = gql`
 `
 export default class UserContactMethodDeleteDialog extends React.PureComponent {
   static propTypes = {
-    cmID: p.string.isRequired,
+    contactMethodID: p.string.isRequired,
+    onClose: p.func.isRequired, // passed to FormDialog
```

Comment:
If a user has two contact methods, and on trying to delete the first contact method (doesn't matter disabled or not), the dialog for deleting contact method does not close on successful deletion. 
This is not observed when there is only one contact method.